Makefile

```diff
@@ -47,12 +49,16 @@ test-java-radio:
 
 test-pyspark-templates:
 	LOCAL_IMAGE=$(S2I_TEST_IMAGE_PYSPARK) make -f Makefile.pyspark build
-	test/e2e/run.sh templates/pyspark
+	test/e2e/run.sh templates/pyspark-py27
 
 test-pyspark-radio:
 	LOCAL_IMAGE=$(S2I_TEST_IMAGE_PYSPARK) make -f Makefile.pyspark build
 	test/e2e/run.sh templates/pyspark/radio
```

Comment:
doesn't affect the tests but I think this changes to templates/pyspark-27/radio doesn't it?

Reply:
oh, good point. once we get this in place, i can change the test template for the site

Reply:
ok, updated the pr once more with the radio changes